_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -168,6 +183,7 @@ cats
 > {: .solution}
 {: .challenge}
 
+#Realistic example
```

Comment:
A agree with your point about the last example being a higher-level summary, but the H2-only convention used throughout all of the SWC lessons is based on how Jekyll and our CSS format the lesson; we ask folks not to use nested headers for this reason. See [here on the lesson-formatting page](http://swcarpentry.github.io/lesson-example/04-formatting/) for more details.